### PR TITLE
Update PPP 

### PR DESCRIPTION
**Pull request by chao890**
Update PPP, almost complete now other than Developer guide contributions, which are still in progress.

### CODE REVIEW

File: docs/team/chao890.md [hunk 1]
```diff
@@ -11,4 +11,59 @@ Given below are my contributions to the project.
...
+    * Created a `DeleteLeaveCommand` class to handle the logic (Refer to code [here](https://github.com/AY2223S1-CS2103T-T15-1/tp/blob/master/src/main/java/coydir/logic/commands/DeleteLeaveCommand.java)) 
+    and `DeleteLeaveCommandParser` class to handle the parsing of the user input (Refer to code [here](https://github.com/AY2223S1-CS2103T-T15-1/tp/blob/master/src/main/java/coydir/logic/parser/DeleteLeaveCommandParser.java)).
+    * Use Case: User can delete leave periods for employees.
+    * Justification: With this command, users of Coydir will be able to make changes easily when employees change their leave periods.
+    * Highlights: Smoother to implement, though just like `add-leave`, there were many dependencies and many files to be edited.
+
+
+* **Existing Features**
+  * Improved on the `Leave` class, by storing `Leave` objects in a priority queue, and adding methods to check for overlapping or equal leaves.
+  * Refactor `Delete` command.
```
Comment (Kok-je) [inline docs/team/chao890.md:38]: Past tense here: "Refactored"

File: docs/team/chao890.md [hunk 2]
```diff
@@ -11,4 +11,59 @@ Given below are my contributions to the project.
...
+  * Implemented `delete-leave` feature
+    * Created a `DeleteLeaveCommand` class to handle the logic (Refer to code [here](https://github.com/AY2223S1-CS2103T-T15-1/tp/blob/master/src/main/java/coydir/logic/commands/DeleteLeaveCommand.java)) 
+    and `DeleteLeaveCommandParser` class to handle the parsing of the user input (Refer to code [here](https://github.com/AY2223S1-CS2103T-T15-1/tp/blob/master/src/main/java/coydir/logic/parser/DeleteLeaveCommandParser.java)).
+    * Use Case: User can delete leave periods for employees.
+    * Justification: With this command, users of Coydir will be able to make changes easily when employees change their leave periods.
+    * Highlights: Smoother to implement, though just like `add-leave`, there were many dependencies and many files to be edited.
+
+
+* **Existing Features**
+  * Improved on the `Leave` class, by storing `Leave` objects in a priority queue, and adding methods to check for overlapping or equal leaves.
```
Comment (Kok-je) [inline docs/team/chao890.md:37]: Consider rephrasing: "Made improvements to `Leave` class by storing..." , "implemented methods to..."

also remove comma after class and queue.

**Review by Kok-je — CHANGES_REQUESTED** [review]
LGTM. Other than some minor phrasing errors. Can change it abit.

**Review by kevinchangjk — APPROVED** [review]
LGTM.

**Review by Kok-je — APPROVED** [review]

**Pull request merged**